frontend/src/site/Login.tsx

```diff
@@ -31,6 +31,31 @@ export default function Login(
     const handlePassword = (e: ChangeEvent<HTMLInputElement>) => {
         setPassword(e.target.value)
     }
+
+    const pwHasLowerLetter = () => {
```

Comment:
Bei den Validatoren wäre es natürlich schon etwas schöner `password` auszuschreiben á la `passwordHasLowerLetter`.
..wobei -> wäre es das? Ist die kürzere Variante nicht besser lesbar?

^Hier ist ein sehr typisches Beispiel bei dem es schwierig zu sagen was besser und was schlechter ist.
Generell ist es heutzutage als besser gesehen Abkürzungen weitgehend zu vermeiden.
In dem Kontext dieses Funktionskörpers kann man natürlich sehen das es sich um ein Passwort handelt -> An anderer Stelle _könnte_ es anfänglich Zweifel geben.

Ohne konkrete Antwort oder Suggeriung mal dieses kleine Essay für dich!

PS: Und was ist mit `hasLowerLetter`? Auch eine schöne Möglichkeit! In manchen Frameworks dann so gehandelt und Lesbar gemacht:

`Validator.hasLowerLetter(password)` <- statische Klasse mit Validatoren